### PR TITLE
Update Googlevideo

### PR DESCRIPTION
**Pull request by ppoozl**
Add vgqsknez q4flrnek a5meknsd a5mlrn7d n4v7sn7y a5mlrnes a5meknsy & some GGC servers
Disable 
vgqs7n7e,vgqs7n7l,vgqs7n7s,vgqs7n7z,vgqs7n76,vgqs7n7d, vgqs7n7k,vgqs7n7r,vgqs7n7y,vgqs7ne7,vgqs7nee,vgqs7nel, vgqs7ne6,vgqs7ned,q4f7sn7y,q4f7sne7,q4f7snee,q4f7snel, q4f7snes,q4f7snez,q4f7sne6,q4f7sned,q4f7snek,q4f7sner,
q4f7sney,q4f7snl7,q4f7snle,q4f7snll,q4f7snls, q4f7snlz,
q4f7snl6,q4f7snld,q4f7snlk,q4f7snlr,q4f7snse, q4f7snsl,
q4f7sns6,q4f7snsd,a5m7ln7s,a5m7ln7d,a5m7ln7z,a5m7ln76,
a5m7ln7d,a5m7ln7k,a5m7ln7r,a5m7ln7y,a5m7lne7,a5m7znee,
a5m7lnee,a5m7znel,a5m7lnel,n4v7kn7e,n4v7kn7l,n4v7kn7s,
n4v7kn7z,n4v7kn7d,n4v7kn7k,n4v7kn7r,n4v7kne7, n4v7knee,n4v7knel

### CODE REVIEW

File: data/ggc.txt [hunk 1]
```diff
@@ -74,6 +78,7 @@ qsc-fra1 9nj-4g5e
 renater-cdg1 gxo5uxg-jqbe
 retelit-mxp1 gxuxapu-hm2e
 ronix-otp1 gvopm-vu2e
+starhub-sin6 sn-nu5gi0c-npod
```
Comment (lennylxx) [inline data/ggc.txt]: remove sn-

File: data/ggc.txt [hunk 2]
```diff
@@ -60,10 +60,14 @@ ixsforall-mnl1 pmn4vg5aa-hoae
 lvlt-arn1 a8au-5goe
 lvlt-bna1 a8au-co5e
 lvlt-bos1 a8au-cvne
+lvlt-dus1 a8au-q0ne
 lvlt-ewr1 a8au-xfge
 lvlt-iad3 a8au-p5qs
 lvlt-iad4 a8au-p5qz
 lvlt-lax1 a8au-a5me
+lvlt-lhr2 a8au-aigl
```
Comment (lennylxx) [inline data/ggc.txt]: I don't see these new ggc servers added in the hosts file, are they available?

File: data/ggc.txt (hunk 1 above)
Comment (ppoozl) [reply data/ggc.txt]: OK

File: data/ggc.txt (hunk 2 above)
Comment (ppoozl) [reply data/ggc.txt]: Sorry for my mistake

Comment (ppoozl) [reply data/ggc.txt]: Line 4885

File: hosts [hunk 3]
```diff
@@ -4842,6 +4858,38 @@
...
+2001:4c08:2010:2::13 r8---sn-a8au-h5ql.googlevideo.com
+2001:4c08:2010:3::c r1---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::d r2---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::e r3---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::f r4---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::10 r5---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::11 r6---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::12 r7---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:3::13 r8---sn-a8au-q0ne.googlevideo.com
+2001:4c08:2010:5::c r1---sn-a8au-aigl.googlevideo.com
```
Comment (ppoozl) [inline hosts]: Here

**Comment by lennylxx** [comment]
Thanks!

**Comment by ppoozl** [comment]
sorry for  inconvenience

**Comment by roc1010kmt** [comment]
支持更新！禁用的那些是怎麼了？

**Pull request merged**